as2_utilities/as2_geozones/tests/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# GTest
```

Comment:
Use documentation CMakeLists.txt for tests

Reply:
Documentation CMakeList do not work